ape_aws/accounts.py

```diff
@@ -1,37 +1,86 @@
 from functools import cached_property
+from json import dumps
+from pathlib import Path
 from typing import Any, Iterator, Optional
 
 from ape.api.accounts import AccountAPI, AccountContainerAPI, TransactionAPI
 from ape.types import AddressType, MessageSignature, SignableMessage, TransactionSignature
+from ape.utils.validators import _validate_account_passphrase
+from eth_account import Account as EthAccount
 from eth_account._utils.legacy_transactions import serializable_unsigned_transaction_from_dict
 from eth_account.messages import _hash_eip191_message, encode_defunct
 from eth_pydantic_types import HexBytes
 from eth_typing import Hash32
-from eth_utils import keccak, to_checksum_address
+from eth_utils import keccak, to_bytes, to_checksum_address
 
 from .client import kms_client
 from .utils import _convert_der_to_rsv
 
 
 class AwsAccountContainer(AccountContainerAPI):
+    loaded_accounts: dict[str, "KmsAccount"] = {}
+
+    def model_post_init(self, __context: Any):
+        print("Initializing AWS KMS Account Container")
+        print([acc.alias for acc in self.accounts])
+
+    @property
+    def _keyfiles(self) -> list[Path]:
+        return [file for file in self.data_folder.glob("*.json")]
+
     @property
     def aliases(self) -> Iterator[str]:
-        return map(lambda x: x.alias, kms_client.raw_aliases)
+        return map(lambda x: x.alias.replace("alias/", ""), kms_client.raw_aliases)
 
     def __len__(self) -> int:
         return len(kms_client.raw_aliases)
 
     @property
     def accounts(self) -> Iterator[AccountAPI]:
+        def _load_account(key_alias, key_id, key_arn) -> AccountAPI:
+            filename = f"{key_alias}.json"
+            keyfile = self.data_folder.joinpath(filename)
+            if filename not in self._keyfiles:
+                self.loaded_accounts[keyfile.stem] = KmsAccount(
+                    key_alias=key_alias,
+                    key_id=key_id,
+                    key_arn=key_arn,
+                )
+                keyfile.write_text(self.loaded_accounts[keyfile.stem].dump_to_json())
+            return self.loaded_accounts[keyfile.stem]
```

Comment:
Don't think we need this, it should just give us what we have in KMS without caching it locally